packages/shortcuts-extension/schema/plugin.json

```diff
@@ -175,6 +175,51 @@
       },
       "type": "object"
     },
+    "imageviewer:rotate-clockwise": {
+      "default": { },
+      "properties": {
+        "command": { "default": "imageviewer:rotate-clockwise" },
+        "keys": { "default": ["["] },
```

Comment:
I think we want to change this line to specify "]", and similarly below for the other shortcut.